products/bitbucket.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Bitbucket
+category: server-app
+tags: atlassian java-runtime
+iconSlug: bitbucket
+permalink: /bitbucket
+alternate_urls:
+-   /atlassian-bitbucket
+releasePolicyLink: https://confluence.atlassian.com/enterprise/atlassian-enterprise-releases-948227420.html#LongTermSupportreleases-Policyanddetails
+changelogTemplate: https://confluence.atlassian.com/display/BitbucketServer/Bitbucket+Data+Center+__RELEASE_CYCLE__+release+notes
+
+eolColumn: Support
+releaseDateColumn: true
+
+identifiers:
+-   cpe: cpe:/a:atlassian:bitbucket
+-   cpe: cpe:2.3:a:atlassian:bitbucket
+
+auto:
+  methods:
+  -   custom: bitbucket
+  -   atlassian_eol: Bitbucket
```

Comment:
```suggestion
```

Unfortunately https://github.com/endoflife-date/release-data/blob/main/src/atlassian_eol.py needs to be adapted to support Bitbucket.